src/uncrustify_emscripten.cpp

```diff
@@ -612,1174 +588,629 @@ intptr_t _debug(intptr_t _file, lang_flag_e langIDX)
 }
 
 
-//! helper function to access option_map_value::name
-string option_map_value_name(const option_map_value &o)
-{
-   return((o.name != nullptr) ? string(o.name) : "");
-}
 
+//template<typename MapType>
```

Comment:
BTW, what is up with all this (new?) commented out code? Is it useful?

Reply:
No, that was just testing stuff, I'll remove it with another commit.